frontend/src/views/ProjectView.vue

```diff
@@ -27,8 +27,10 @@
 
 <script setup lang="ts">
 import ProjectInfo from "@/components/project/ProjectInfo.vue";
+
 import { useProjectQuery } from "@/queries/Project";
 import { computed, toRefs } from "vue";
+import NeedHelpButton from "@/components/buttons/NeedHelpButton.vue";
```

Comment:
```suggestion
import NeedHelpButton from "@/components/buttons/NeedHelpButton.vue";
```
knop wordt hier niet gebruikt

Reply:
hoe is da der zelfs bijgekomen lol kheb da nie aangeraakt afaik